README.md

```diff
@@ -11,31 +11,31 @@
-[![Revved up by Gradle Enterprise](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.solutions-team.gradle.com/scans)
+[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.solutions-team.gradle.com/scans)
 
-The Common Custom User Data Maven extension for Gradle Enterprise enhances published build scans
-by adding a set of tags, links and custom values that have proven to be useful for many projects building with Gradle Enterprise.
+The Common Custom User Data Maven extension for Develocity enhances published build scans
+by adding a set of tags, links and custom values that have proven to be useful for many projects building with Develocity.
 
 You can leverage this extension for your project in one of two ways:
 1. [Apply the published extension](#applying-the-published-extension) directly in your `.mvn/extensions.xml` and immediately benefit from enhanced build scans
-2. Copy this repository and [develop a customized version of the extension](#developing-a-customized-version-of-the-extension) to standardize Gradle Enterprise usage across multiple projects
+2. Copy this repository and [develop a customized version of the extension](#developing-a-customized-version-of-the-extension) to standardize Develocity usage across multiple projects
 
 ## Applying the published extension
 
 The Common Custom User Data Maven extension is available in [Maven Central](https://search.maven.org/artifact/com.gradle/common-custom-user-data-maven-extension). This extension
-requires the [Gradle Enterprise Maven extension](https://search.maven.org/artifact/com.gradle/gradle-enterprise-maven-extension) to also be applied in your build in order to have
+requires the [Develocity Maven extension](https://search.maven.org/artifact/com.gradle/gradle-enterprise-maven-extension) to also be applied in your build in order to have
 an effect.
 
 In order for the Common Custom User Data Maven extension to become active, you need to register it in the `.mvn/extensions.xml` file in your root project. The `extensions.xml` file
-is the same file where you have already declared the Gradle Enterprise Maven extension.
+is the same file where you have already declared the Develocity Maven extension.
 
 See [here](.mvn/extensions.xml) for an example.
 
 ### Version compatibility
 
-This table details the version compatibility of the Common Custom User Data Maven extension with the Gradle Enterprise Maven extension.
+This table details the version compatibility of the Common Custom User Data Maven extension with the Develocity Maven extension.
 
-| Common Custom User Data Maven extension versions | Gradle Enterprise Maven extension versions |
+| Common Custom User Data Maven extension versions | Develocity Maven extension versions |
```

Comment:
Can you reformat the table to account for the shorter column header?

Reply:
good catch!